site/package.json

```diff
@@ -106,6 +106,7 @@
     "@storybook/addon-links": "7.5.2",
     "@storybook/addon-mdx-gfm": "7.5.2",
     "@storybook/addon-themes": "7.6.4",
+    "@storybook/preview-api": "7.6.9",
```

Comment:
What does it do?

Reply:
This is for the `useArgs` hook I'm using to update state of a rendered component in storybook.